File: Leetcode/172-Factorial-Trailing-Zeroes/FactorialTrailingZeroes.py
class Solution(object):
    def trailingZeroes(self, n):
        """
        :type n: int
        :rtype: int
        """
        if n<0 :
            return -1
        ans = 0
        while n>0 :
            n = n/5
            ans += n
        return ans
        

File: Leetcode/172-Factorial-Trailing-Zeroes/FactorialTrailingZeroes.cpp
class Solution {
public:
    int trailingZeroes(int n) {
        if (n<0) {
            return -1;
        }
        
        int ans = 0;
        while (n>0) {
            n /= 5;
            ans += n;
        }
        return ans;
    }
};


